external/config/kernel/linux-rockchip-rk3588-current.config

```
@@ -2290,7 +2290,7 @@ CONFIG_SATA_AHCI_PLATFORM=y
 CONFIG_MD=y
 # CONFIG_BLK_DEV_MD is not set
 # CONFIG_BCACHE is not set
-# CONFIG_BLK_DEV_DM is not set
+CONFIG_BLK_DEV_DM=y
```

I suggest compiling this directly into the kernel by default.  Alternately, if you don't want this it can be included as a loadable module.

```suggestion
CONFIG_BLK_DEV_DM=m
```